src/main/resources/static/js/AlbumFront.js

```diff
@@ -0,0 +1,117 @@
+function getIndex(list, id) {
+    for (var i = 0; i < list.length; i++) {
+        if (list[i].id === id) {
+            return i;
+        }
+    }
+
+    return -1;
+}
+
+var albumApi = Vue.resource("/albums/{/id}");
+
+Vue.component('album-form', {
```

Comment:
Please don't use frameworks. Should use vanilla js or don't write a frontend at all